ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/services/FragmentCreator.java

```diff
@@ -1,4 +1,4 @@
-package be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.services.fragmentation;
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.services;
```

Comment:
Why this got moved out fragmentation is a mystery to me. The Ldes Server will for sure grow so compartmenting doesn't hurt.